pkg/network/vmispec/interface_test.go

```diff
@@ -238,7 +238,18 @@ var _ = Describe("VMI network spec", func() {
 			deviceInfoPlugin:    {DownwardAPI: v1.DeviceInfo},
 			nonDeviceInfoPlugin: {},
 		}
-
+		It("returns false when interface binding is not plugin with device-info", func() {
```

Comment:
The context is related to method `BindingPluginNetworkWithDeviceInfoExist`. The two added tests should have a separate context.

Reply:
One function depend on the other and duplicating the context with its vars and consts seems redundent.
Can we have an exception for those tests?

Reply:
Done

The const and vars were moved outside the original context to enable reuse.

Reply:
?